docs/CLI.md

```diff
@@ -318,6 +318,21 @@ If configuration files are found in the specified paths, _all_ projects specifie
 
 :::
 
+### `--randomize`
+
+Shuffle the order of the tests within a file. The seed used to generate the new order will be printed to stdout.
+In order to determine the seed refer to the [`--seed`](#--seed) CLI option.
+
+```bash
+jest --randomize
+```
```

Comment:
What you think about this? (Ignore / remove extra space in code block, I have to use them to make the suggestion work here.)

```suggestion

:::tip
Additionally you can use the [`--seed`](#--seed) flag to pass the seed value:

    ```bash
    jest --randomize --seed 1234
    ```
:::
```